Signal/src/ViewControllers/FingerprintViewController.m

```diff
@@ -20,34 +21,46 @@
 
 @interface FingerprintViewController () <OWSHighlightableLabelDelegate, OWSCompareSafetyNumbersActivityDelegate>
 
-@property (strong, nonatomic) TSStorageManager *storageManager;
-@property (strong, nonatomic) OWSFingerprint *fingerprint;
-@property (strong, nonatomic) NSString *contactName;
-@property (strong, nonatomic) OWSQRCodeScanningViewController *qrScanningController;
-
-@property (strong, nonatomic) IBOutlet UINavigationBar *modalNavigationBar;
-@property (strong, nonatomic) IBOutlet UIBarButtonItem *dismissModalButton;
-@property (strong, nonatomic) IBOutlet UIBarButtonItem *shareButton;
-@property (strong, nonatomic) IBOutlet UIView *qrScanningView;
-@property (strong, nonatomic) IBOutlet UILabel *scanningInstructions;
-@property (strong, nonatomic) IBOutlet UIView *scanningContainer;
-@property (strong, nonatomic) IBOutlet UIView *instructionsContainer;
-@property (strong, nonatomic) IBOutlet UIView *qrContainer;
-@property (strong, nonatomic) IBOutlet UIView *privacyVerificationQRCodeFrame;
-@property (strong, nonatomic) IBOutlet UIImageView *privacyVerificationQRCode;
-@property (strong, nonatomic) IBOutlet OWSHighlightableLabel *privacyVerificationFingerprint;
-@property (strong, nonatomic) IBOutlet UILabel *instructionsLabel;
-@property (strong, nonatomic) IBOutlet UIButton *scanButton;
+@property (nonatomic) TSStorageManager *storageManager;
+@property (nonatomic) OWSFingerprint *fingerprint;
+@property (nonatomic) NSString *contactName;
+@property (nonatomic) OWSQRCodeScanningViewController *qrScanningController;
+
+@property (nonatomic) IBOutlet UINavigationBar *modalNavigationBar;
```

Comment:
shucks, I was hoping if I dragged my feet you'd turn this into a programmatic VC in no time.